packages/mdc-fab/mdc-fab.scss

```diff
@@ -22,7 +22,7 @@
   @include mdc-fab-base_;
   @include mdc-fab-container-color(secondary);
   @include mdc-fab-ink-color(text-primary-on-secondary);
-  @include mdc-ripple-color(text-primary-on-secondary, $mdc-ripple-pressed-dark-ink-opacity);
+  @include mdc-states(text-primary-on-secondary);
 
   &:not(.mdc-ripple-upgraded) {
     -webkit-tap-highlight-color: rgba(0, 0, 0, .18);
```

Comment:
Is there are reason this remains a literal value and does not use the mdc-theme-prop() mixin?

Reply:
mdc-states already uses mdc-theme-prop-value internally so you can pass a theme property here. mdc-ripple-color used its own logic analogous to that.

Reply:
Ah, I was referring to line 28

Reply:
Oh. Actually I'm not sure why we never outright removed that style, since we removed it from button styles several releases ago, and it actually looks super-awkward for FAB (it creates a rectangular shade over the FAB when the FAB is round). So I've removed it.